### PR TITLE
Release Note for the enterprise_sonic v2.2.0

### PR DESCRIPTION
**Pull request by kerry-meyer**
##### SUMMARY

Provide release notes for the 2.2.0 version of the enterprise_sonic Ansible Network Resource Module Collection

- Provide a summary fragment to generate the release note summary.
- Fix non-conformant old fragments in the changelogs/changelog.yaml to generate correct CHANGELOG.rst output for this release and for future releases.

##### GitHub Issues
List the GitHub issues impacted by this PR. If no Github issues are affected, please indicate this with "N/A".

| GitHub Issue # |
| -------------- |
| |
N/A

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+release_summary: |
+        | Release Date: 2023-0531
+        | This release provides Ansible compliance changes required on top of the changes included in
+        | the 2.1.0 release of the enterprise_sonic Ansible network resource module collection.
+        | It addresses two issues raised by the Ansible core team with the content of the 2.1.0 release.
+        | 1) Back out the "breaking_change" made in the sonic_aaa resource module to to fix a functional
```
Comment (ArunSaravananBalachandran) [inline changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml:6]:
```suggestion
        | 1) Back out the "breaking_change" made in the sonic_aaa resource module to fix a functional
```

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -100,63 +100,57 @@ releases:
...
-        (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/90)
-      - aaa - Modify regression test sequencing to enable correct testing of the functionality
-        for this module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/78).
-      - bgp_neighbors - remove string conversion of timer attributes (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/60)
+      - bgp_neighbors - Remove string conversion of timer attributes (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/60).
       - port_breakout - Fixed a bug in formulation of port breakout REST APIs (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/88).
       - sonic - Fix a bug in handling of interface names in standard interface naming
         mode (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/103).
+        branching (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/90).
+      - sonic_aaa - Fix a bug in facts gathering by providing required conditional
```
Comment (ArunSaravananBalachandran) [inline changelogs/changelog.yaml:128]:
```suggestion
      - sonic_aaa - Fix a bug in facts gathering by providing required conditional
      branching (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/90).
```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -455,19 +450,69 @@ releases:
...
+        changes included in
+
+        | the 2.1.0 release of the enterprise_sonic Ansible network resource module
+        collection.
+
+        | It addresses two issues raised by the Ansible core team with the content
+        of the 2.1.0 release.
+
+        | 1) Back out the "breaking_change" made in the sonic_aaa resource module
+        to to fix a functional
```
Comment (ArunSaravananBalachandran) [inline changelogs/changelog.yaml:487]:
```suggestion
        to fix a functional
```

**Review by ArunSaravananBalachandran — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,34 @@ Dellemc.Enterprise_Sonic Release Notes
...
+======
+
+Release Summary
+---------------
+
+| Release Date: 2023-0531
+| This release provides Ansible compliance changes required on top of the changes included in
+| the 2.1.0 release of the enterprise_sonic Ansible network resource module collection.
+| It addresses two issues raised by the Ansible core team with the content of the 2.1.0 release.
+| 1) Back out the "breaking_change" made in the sonic_aaa resource module to to fix a functional
```
[the remark this answers is not in the document]
Comment (kerry-meyer) [reply CHANGELOG.rst:18]: I have made this change in the ".yaml" file.

File: changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml (hunk 1 above)
Comment (kerry-meyer) [reply changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml:6]: I manually edited this before seeing this comment.

File: changelogs/changelog.yaml (hunk 2 above)
Comment (kerry-meyer) [reply changelogs/changelog.yaml:128]: Same here.

File: changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml (hunk 1 above)
Comment (kerry-meyer) [reply changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml:6]: Fragment files are not used after they are incorporated into the changelog.yaml file and archived. The CHANGELOG.rst file is re-generated based solely on the changelog.yaml file and does not refer to archived fragments.

I am accepting this change to resolve the issue, but in the future, it is not necessary to check or correct archived fragment files when reviewing release notes.

File: changelogs/changelog.yaml (hunk 2 above)
Comment (kerry-meyer) [reply changelogs/changelog.yaml:128]: Fragment files are not used after they are incorporated into the changelog.yaml file and archived. The CHANGELOG.rst file is re-generated based solely on the changelog.yaml file and does not refer to archived fragments.

I am accepting this change to resolve the issue, but in the future, it is not necessary to check or correct archived fragment files when reviewing release notes.

File: changelogs/changelog.yaml (hunk 3 above)
Comment (kerry-meyer) [reply changelogs/changelog.yaml:487]: Same here.

**Pull request merged**